src/Servers/IIS/IIS/src/Core/IISHttpContext.FeatureCollection.cs

```diff
@@ -199,14 +199,26 @@ Task IHttpResponseBodyFeature.SendFileAsync(string path, long offset, long? coun
 
         // TODO: In the future this could complete the body all the way down to the server. For now it just ensures
         // any unflushed data gets flushed.
-        protected Task CompleteResponseBodyAsync()
+        protected async Task CompleteResponseBodyAsync()
         {
             if (ResponsePipeWrapper != null)
             {
-                return ResponsePipeWrapper.CompleteAsync().AsTask();
+                await ResponsePipeWrapper.CompleteAsync().AsTask();
```

Comment:
Pretty sure this isn't needed
```suggestion
                await ResponsePipeWrapper.CompleteAsync();
```